lms/templates/courseware/dates.html

```diff
@@ -95,3 +95,17 @@ <h2 class="hd hd-2 date-title">
 <%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
     DateUtilFactory.transform(iterationKey=".localized_datetime");
 </%static:require_module_async>
+
+<%static:require_module_async module_name="js/commerce/track_ecommerce_events" class_name="TrackECommerceEvents">
+  var PLSUpgradeLink = $("#PLS_button");
+
+function trackPLSUpgradeEvent() {
+    TrackECommerceEvents.trackUpsellClick(PLSUpgradeLink, 'dates_upgrade', {
+      pageName: "dates_tab",
+      linkType: "button",
+      linkCategory: "PLS_button"
+    });
+  }
+  PLSUpgradeLink.unbind('click.ecommerce_link').bind('click.ecommerce_link', trackPLSUpgradeEvent);
```

Comment:
Is the JS on this page loaded twice as well? The trackUpsellClick method should be doing all of the event binding, so you'll probably need to set the state on the window variable (see https://github.com/edx/edx-platform/pull/24338/commits/ae33c23e562cd64ea4b60c1b19d55cbbcfe54417)

Reply:
Good catch! This was patterned after the initial code- I'll make the similar changes as the commit you linked- thanks!

Reply:
Added!